src/toxcore/state_format/old.rs

```diff
@@ -525,6 +526,7 @@ impl FromBytes for FriendState {
 }
 
 impl ToBytes for FriendState {
+    #[allow(clippy::cyclomatic_complexity)]
```

Comment:
That's not a fix)

Reply:
But it's a false positive warning due to macro expansion. By definition, cyclomatic complexity is a quantitative measure of the number of linearly independent paths through a program's source code. Obviously, this this method has only one path, which means it's cyclomatic complexity is 1.